build.gradle.kts

```diff
@@ -131,6 +142,36 @@ tasks {
 		group = MAIN
 	}
 	
+	val buildInstaller by creating(Install4jTask::class) {
+		dependsOn(shadowJar)
+		group = MAIN
+		
+		var path : File? = null
+		doFirst {
+			println("[INSTALL4J] Installers will be saved in ./install4j directory")
```

Comment:
please put the install4j directory under `build`, because it is part of the build output

Reply:
ef8587656e17d6e7fb927f12874a1b01153d9f78